packages/design-system/src/components/analytics/getRenderedTextContent.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOMServer from 'react-dom/server';
+
+export function getRenderedTextContent(reactNode: string | React.ReactNode): string {
```

Comment:
This is the second idea, and I show how it would be implemented in the `HelpDrawer` component

Reply:
I guess this works and is simple, but it feels like a slight mis-use of this library which seems to be intended to be used on a server 🤷🏻‍♀️

Reply:
That was my gut reaction as well, but if it works well and is simple I thought I'd put that feeling aside.. I am curious how it will be used in the wild, what an actual use case would look like

Reply:
The 2nd idea has a cleaner implementation that I like more than the first idea